pkg/controllers/status/controller.go

```diff
@@ -276,22 +279,27 @@ func (s *KubeFedStatusController) reconcile(qualifiedName common.QualifiedName)
 	targetIsDeployment := schemautil.APIResourceToGVK(&targetType) == appsv1.SchemeGroupVersion.WithKind(common.DeploymentKind)
 	statusKind := s.typeConfig.GetStatusType().Kind
 	key := qualifiedName.String()
+	keyedLogger := s.logger.WithValues("object", key)
+	ctx := klog.NewContext(context.Background(), keyedLogger)
 
 	s.metrics.Rate("status.throughput", 1)
-	klog.V(4).Infof("Starting to reconcile %v %v", statusKind, key)
+	keyedLogger.WithValues("status-kind", statusKind).V(3).Info("Starting reconcile")
 	startTime := time.Now()
 	defer func() {
 		s.metrics.Duration("status.latency", startTime)
-		klog.V(4).Infof("Finished reconciling %v %v (duration: %v)", statusKind, key, time.Since(startTime))
+		keyedLogger.WithValues("duration", time.Since(startTime), "status-kind", statusKind).
+			V(3).Info("Finished reconcile")
 	}()
 
-	fedObject, err := s.objFromCache(s.federatedStore, federatedKind, key)
+	fedObject, err := s.objFromCache(ctx, s.federatedStore, federatedKind, key)
 	if err != nil {
 		return worker.StatusError
 	}
 
 	if fedObject == nil || fedObject.GetDeletionTimestamp() != nil {
-		klog.V(4).Infof("No federated type for %v %v found, about to delete status object", federatedKind, key)
+
+		keyedLogger.WithValues("federated-kind", federatedKind).
+			V(3).Info("No federated type found, about to delete status object")
```

Comment:
This should be v1 since it reconciles deleted federated objects and is also an effectual action

Reply:
done, pls take review.